packages/sanity/src/core/store/_legacy/document/document-pair/checkoutPair.ts

```diff
@@ -60,18 +60,17 @@ export interface DocumentVersion {
 /**
  * @hidden
  * @beta */
-// TODO: Rename -> `Bundle`
```

Comment:
Did you settle on not renaming to bundle?

Reply:
Not 100% sure, but I'm leaning towards not renaming. For a couple of reasons:

- The publish and draft documents are always included in the pair. Conceptually, this structure can still be thought of as a *pair*, with one additional (optional) slot for the checked-out version.
- Renaming would touch a lot of surfaces. It's not strictly a problem to do so, but I think it's nice to minimise this churn if we don't have a good reason for it.

I'm happy to rename it if we think that's best, though! "Group" would certainly be more accurate, but it is worth the churn and breaking something folks are already familiar with?